utils/tests/utils.go

```diff
@@ -302,6 +304,44 @@ func DeleteFiles(deleteSpec *spec.SpecFiles, artifactoryDetails *config.Artifact
 	return deleteCommand.DeleteFiles(reader)
 }
 
+func GetBuildInfo(t *testing.T, artDetails *config.ArtifactoryDetails, buildName, buildNumber string) (buildinfo.BuildInfo, error) {
+	servicesManager, err := artUtils.CreateServiceManager(artDetails, false)
+	if err != nil {
+		assert.NoError(t, err)
+		return buildinfo.BuildInfo{}, err
+	}
+	params := services.NewBuildInfoParams()
+	params.BuildName = buildName
+	params.BuildNumber = buildNumber
+	bi, notFound, err := servicesManager.GetBuildInfo(params)
+	if err != nil {
+		assert.NoError(t, err)
+		return buildinfo.BuildInfo{}, err
+	}
+	if notFound {
+		assert.False(t, notFound, "expected to find build info")
+		return buildinfo.BuildInfo{}, errors.New("expected to find build info")
+	}
+	return bi.BuildInfo, nil
+}
+
+func VerifyBuildInfoDoesntExist(t *testing.T, artDetails *config.ArtifactoryDetails, buildName, buildNumber string) (doesntExist bool, err error) {
```

Comment:
This function verifies that the build info requested does not exist. Looking at this function's two usages, they also do the same check, by using the *doesntExist* returned value. I actually suggest merging this function with its neighbouring *GetBuildInfo* function. The *GetBuildInfo* can simply return the *doesntExist* boolean, and not do any assets. The calling functions can do the assets.